css/index.css

```diff
@@ -143,4 +218,23 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+    font-weight: bold;
+}
+
+@keyframes shake {
```

Comment:
This is an elaborate animation - did you take this off the internet or did you write it all out yourself?